### ISSUE TITLE
[release-tests] ml_user_ray_lightning_user_test_(master|latest).aws failed

### ISSUE THREAD

**Issue by can-anyscale** [ray-project/ray#35437]
### What happened + What you expected to happen

ml_user_ray_lightning_user_test_master.aws and ml_user_ray_lightning_user_test_latest.aws  are consistently failing

### Versions / Dependencies

https://buildkite.com/ray-project/release-tests-branch/builds/1657#0188286a-6c87-4266-8368-1cd7257056bd

### Reproduction script

https://buildkite.com/ray-project/release-tests-branch/builds/1657#0188286a-6c87-4266-8368-1cd7257056bd

### Issue Severity

High: It blocks me from completing my task.

**Comment by can-anyscale** [comment]
This is related to urllib3 dependency issues

**Comment by woshiyyya** [comment]
hmm weird. There are no logs in the anyscale job page.

By the way, maybe we can remove the release test of ray_lightning? @matthewdeng

**Comment by can-anyscale** [comment]
Here is the stack trace from the buildkite job. It's urllib3 issue.

```
Traceback (most recent call last):
--
  | File "/home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/modules/runtime_env/runtime_env_agent.py", line 352, in _create_runtime_env_with_retry
  | runtime_env_setup_task, timeout=setup_timeout_seconds
  | File "/home/ray/anaconda3/lib/python3.7/asyncio/tasks.py", line 442, in wait_for
  | return fut.result()
  | File "/home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/modules/runtime_env/runtime_env_agent.py", line 312, in _setup_runtime_env
  | runtime_env, plugin, uri_cache, context, per_job_logger
  | File "/home/ray/anaconda3/lib/python3.7/site-packages/ray/_private/runtime_env/plugin.py", line 254, in create_for_plugin_if_needed
  | size_bytes = await plugin.create(uri, runtime_env, context, logger=logger)
  | File "/home/ray/anaconda3/lib/python3.7/site-packages/ray/_private/runtime_env/working_dir.py", line 156, in create
  | uri, self._resources_dir, self._gcs_aio_client, logger=logger
  | File "/home/ray/anaconda3/lib/python3.7/site-packages/ray/_private/runtime_env/packaging.py", line 698, in download_and_unpack_package
  | "bucket. " + install_warning
  | ImportError: You must `pip install smart_open` and `pip install boto3` to fetch URIs in s3 bucket. Note that these must be preinstalled on all nodes in the Ray cluster; it is not sufficient to install them in the runtime_env.
  | 2023-05-17 08:01:39,162	ERROR runtime_env_agent.py:359 -- Failed to create runtime env {"env_vars": {"RAY_EXPERIMENTAL_NOSET_CUDA_VISIBLE_DEVICES": "1", "RAY_worker_niceness": "0"}, "working_dir": "s3://ray-release-automation-results/working_dirs/ml_user_ray_lightning_user_test_master.aws/unolrkvfhs/_anyscale_pkg_ffab2b134b6346b4b7d0ad7076f90fc6.zip"}.
  | Traceback (most recent call last):
  | File "/home/ray/anaconda3/lib/python3.7/site-packages/ray/_private/runtime_env/packaging.py", line 692, in download_and_unpack_package
  | import boto3
  | File "/home/ray/anaconda3/lib/python3.7/site-packages/boto3/__init__.py", line 17, in <module>
  | from boto3.session import Session
  | File "/home/ray/anaconda3/lib/python3.7/site-packages/boto3/session.py", line 17, in <module>
  | import botocore.session
  | File "/home/ray/anaconda3/lib/python3.7/site-packages/botocore/session.py", line 26, in <module>
  | import botocore.client
  | File "/home/ray/anaconda3/lib/python3.7/site-packages/botocore/client.py", line 15, in <module>
  | from botocore import waiter, xform_name
  | File "/home/ray/anaconda3/lib/python3.7/site-packages/botocore/waiter.py", line 18, in <module>
  | from botocore.docs.docstring import WaiterDocstring
  | File "/home/ray/anaconda3/lib/python3.7/site-packages/botocore/docs/__init__.py", line 15, in <module>
  | from botocore.docs.service import ServiceDocumenter
  | File "/home/ray/anaconda3/lib/python3.7/site-packages/botocore/docs/service.py", line 14, in <module>
  | from botocore.docs.client import ClientDocumenter, ClientExceptionsDocumenter
  | File "/home/ray/anaconda3/lib/python3.7/site-packages/botocore/docs/client.py", line 17, in <module>
  | from botocore.docs.example import ResponseExampleDocumenter
  | File "/home/ray/anaconda3/lib/python3.7/site-packages/botocore/docs/example.py", line 13, in <module>
  | from botocore.docs.shape import ShapeDocumenter
  | File "/home/ray/anaconda3/lib/python3.7/site-packages/botocore/docs/shape.py", line 19, in <module>
  | from botocore.utils import is_json_value_header
  | File "/home/ray/anaconda3/lib/python3.7/site-packages/botocore/utils.py", line 37, in <module>
  | import botocore.httpsession
  | File "/home/ray/anaconda3/lib/python3.7/site-packages/botocore/httpsession.py", line 22, in <module>
  | from urllib3.util.ssl_ import (
  | ImportError: cannot import name 'DEFAULT_CIPHERS' from 'urllib3.util.ssl_' (/home/ray/anaconda3/lib/python3.7/site-packages/urllib3/util/ssl_.py)
```

**Comment by woshiyyya** [comment]
Thanks @can-anyscale ! I've created a PR and see if the fix works.

**Comment by matthewdeng** [comment]
Should these be pinned in the Docker image instead? cc @krfricke

**Comment by can-anyscale** [comment]
I think the Docker image already has urllib3 1.x.x (there is a fix on anyscale side for this), but I **think** some of the pip install in the cluster environment converts it back to 2.x.x

**Comment by woshiyyya** [comment]
I've reinstalled urllib3 < 1.27, but it seems that the release tests fail on aws but succeed on gce.

https://buildkite.com/ray-project/release-tests-pr/builds/38971#_

**Comment by woshiyyya** [comment]
Okey, fixed by pining urllib < 1.27 as the last post build command.

2 PRs:
- For master branch: https://github.com/ray-project/ray/pull/35465
- For releases/2.5.0 branch: https://github.com/ray-project/ray/pull/35489
